src/HashBus.Twitter.Monitor/TweetMapper.cs

```diff
@@ -40,15 +49,6 @@ private static Tweet MapTweet(ITweet tweet, string track)
                                 ScreenName = userMention.ScreenName,
                             })
                         .ToList(),
-                    Hashtags = tweet.Hashtags
-                        .Select(hashtag =>
-                            new Hashtag
-                            {
-                                Text = hashtag.Text,
-                                Indices = hashtag.Indices,
-                            })
-                        .ToList(),
-                    RetweetedTweet = MapTweet(tweet.RetweetedTweet, track),
                 };
         }
     }

```

Comment:
These changes align the property order with that used in `Application.TweetMapper`.